osquery/tables/system/linux/python_packages.cpp

```diff
@@ -0,0 +1,85 @@
+/*
+*  Copyright (c) 2014-present, Facebook, Inc.
+*  All rights reserved.
+*
+*  This source code is licensed under the BSD-style license found in the
+*  LICENSE file in the root directory of this source tree. An additional grant
+*  of patent rights can be found in the PATENTS file in the same directory.
+*
+*/
+
+#include <string>
+#include <fstream>
+
+#include <osquery/filesystem.h>
+#include <osquery/logger.h>
+#include <osquery/system.h>
+#include <osquery/tables.h>
+
+#include <boost/algorithm/string/split.hpp>
```

Comment:
As for imports, we use an include order of the most broad headers to the most specific, so you'd want
```
  #include <string>
  #include <fstream>

  #include <boost/algorithm/string/split.hpp>
  #include <boost/algorithm/string/trim.hpp>
 
  #include <osquery/filesystem.h>
  #include <osquery/logger.h>
  #include <osquery/system.h>
  #include <osquery/tables.h>
```